application/views/admin/globalsettings/_security.php

```diff
@@ -188,15 +194,21 @@
                     <?php eT("Maximum number of attempts:"); ?>
                 </label>
                 <div class="">
-                    <input class="form-control" min="0" type="number" name="maxLoginAttemptParticipants" value="<?= Yii::app()->getConfig('maxLoginAttemptParticipants') ?>" />
+                    <input class="form-control" type="number" min="1" step="1" pattern="^\d*$" name="maxLoginAttemptParticipants" placeholder="<?= gT("Disabled") ?>"
+                        value="<?= App()->getConfig('maxLoginAttemptParticipants') !== "" ??  intval(App()->getConfig('maxLoginAttemptParticipants')) ?>"
+                    />
+                    <div class="form-text"><?= gT("Set an empty value disable brute force protection. Number of attemps are never checked.") ?></div>
```

Comment:
Please write this sentence like this:
"Set an empty value to disable brute force protection. Number of attempts are never checked."

Reply:
_Set an empty value disable brute force protection. Number of attempts are never checked._

Reply:
Fixed by _Set an empty value to disable brute force protection.  Number of attempts are never checked._